src/content/configuration/dev-server.md

```diff
@@ -1192,6 +1192,23 @@ module.exports = {
 };
 ```
 
+## `devServer.sockPort`
+
+`number` `string`
```

Comment:
Maybe we should add null?

https://github.com/webpack/webpack-dev-server/blob/master/lib/options.json#L239

Reply:
`null` doesn't affect anything, it should be removed from the schema imho:
https://github.com/webpack/webpack-dev-server/blob/cf4d0d07b8ffba9d28cd627b48f74e5f4f307fc6/lib/utils/addEntries.js#L20